roadblock.py

```diff
@@ -215,6 +213,13 @@ def set_message_validation(self, msg_val):
 
         return 0
 
+    def set_connection_watchdog(self, con_watchdog_state):
+        '''Enable/disable the connection watchdog'''
+
+        self.connection_watchdog_state = con_watchdog_state
```

Comment:
You must have a good reason for creating this function. 
If you are creating a function only for a variable assignment, you could also do one or more of the following:
- get the actual var value 
- print debug info with the var value
- check if value is valid
- handle possible errors or unexpected situations